debian/changelog

```diff
@@ -1,5 +1,3 @@
 rdma-core (12-1) unstable; urgency=low
 
-  * New version
-
```

Comment:
Put your initial commit here with the ITP bug and so forth, that will silence Lintian for upstream:

```
W: libibcm1: new-package-should-close-itp-bug
```